lookout/style/format/tests/top_repos_quality.py

```diff
@@ -0,0 +1,199 @@
+"""Measure quality on several top repositories."""
+from argparse import ArgumentParser
+import logging
+import glob
+import io
+import os
+import subprocess
+import tempfile
+
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.core.test_helpers.server import fetch, find_port, run as launch_server
+from lookout.style.format.tests.test_analyzer_integration import TestAnalyzer
+
+# format: "repository to-commit from-commit"
+REPOSITORIES = """https://github.com/30-seconds/30-seconds-of-code 3a122c9cfcbdc091227879a06a32bc67ccd0d35d c8c60895e80b8bc90583502accdaa339b794609c
```

Comment:
let's store it in some `csv` file in https://github.com/src-d/style-analyzer/tree/master/lookout/style/format/benchmarks/data to not clutter up a code. 

Also, we have `benchmark` directory, let's move code related to your benchmark there.

Reply:
I moved code to benchmarks, but I think that separating repositories to `csv` is not necessary - this script is only to generate a report for release (and maybe to check quality for massive changes) on selected top repositories to avoid quality degradation.

Reply:
Maybe, but I like the idea of having a csv file and feeding it into your cmd entry point.
it is clearer what actually happening when you run it.
But as you wish.